packages/terra-navigation-layout/tests/wdio/navigation-layout-spec.js

```diff
@@ -0,0 +1,41 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge');
+const selector = '#test-root';
```

Comment:
At this point, we should need to add a selector for these tests. The default should work fine.

Reply:
The default selector does not work since React 16 upgrade. This error occurs: 
```
boundingRects.reduce is not a function
running chrome
TypeError: boundingRects.reduce is not a function
    at execute(<Function>) - selectorExecute.js:58:65
    at execute("return !!document.evaluate;") - ensureClientSideSelectorSupport.js:23:17
```

Reply:
You can update it in your wdio.conf.js file like so. (https://github.com/cerner/terra-core/blob/master/wdio.conf.js#L43). Will that work for you?

And then once it's on the conf file, I believe you can remove this selector.

Reply:
@JakeLaCombe we have a [default selector](https://github.com/cerner/terra-framework/blob/master/wdio.conf.js#L34) specified in the wdio.config. I have found there are instances when content is 'hidden' or animating off the page and it appears the element is larger than the document size which throws this error. Given we want the screenshot of the entire page, I feel this selector is fine in this instance.